static/js/filter.js

```diff
@@ -231,7 +232,7 @@ Filter.parse = function (str) {
     var operand;
     var term;
 
-    var matches = str.match(/"[^"]+"|\S+/g);
+    var matches = str.match(/(\S+:)?"[^"]+"?|\S+/g);
```

Comment:
Maybe we can improve the regex here to even support something like `stream: "foo bar"`. Basically notice the `space` after typing `stream:`. Currently it just won't work if I put in a space after the column.